### PR TITLE
Remove mdm_ips, add taint msg and check SDC existing name

### PR DESCRIPTION
**Pull request by rounak-adhikary**
Tests
```
root@lglap049:~/terraform-provider-powerflex# go test ./powerflex/provider/ -v -timeout 8h -run TestAccSDCHostResource
=== RUN   TestAccSDCHostResourceUbuntu
--- PASS: TestAccSDCHostResourceUbuntu (21.61s)
=== RUN   TestAccSDCHostResourceEsxiNeg
--- PASS: TestAccSDCHostResourceEsxiNeg (0.32s)
=== RUN   TestAccSDCHostResourceEsxi
--- PASS: TestAccSDCHostResourceEsxi (118.78s)
=== RUN   TestAccSDCHostResourceNegative
--- PASS: TestAccSDCHostResourceNegative (31.55s)
=== RUN   TestAccSDCHostResourcePositive
    sdc_host_resource_win_test.go:58: Skipping this test case for real environment
--- SKIP: TestAccSDCHostResourcePositive (0.00s)
=== RUN   TestAccSDCHostResourceRPMPositive
    sdc_host_resource_win_test.go:83: Skipping this test case for real environment
--- SKIP: TestAccSDCHostResourceRPMPositive (0.00s)
PASS
ok      terraform-provider-powerflex/powerflex/provider 172.293s
root@lglap049:~/terraform-provider-powerflex#
```

### CODE REVIEW

**Comment by terraform-providers-svc** [comment]
Can one of the admins verify this patch?

File: powerflex/helper/sdc_host.go [hunk 1]
```diff
@@ -74,25 +73,16 @@ func (r *SdcHostResource) getSSHProvisioner(ctx context.Context, plan models.Sdc
 	return prov, dir, err
 }
 
-// GetMdmIps - get mdm ips from plan or from pflex
+// GetMdmIps - get mdm ips from pflex
```
Comment (shenda1) [inline powerflex/helper/sdc_host.go:76]: PowerFlex

File: templates/resources/sdc_host.md.tmpl [hunk 2]
```diff
@@ -26,13 +26,20 @@ description: |-
 
 {{ .Description | trimspace }}
 
+~> **Caution:** SDC Host creation is not atomic. This resource sets parameters like name, etc. after SDC installation is complete.
+If that fails for any reason, Terraform, by default, will mark this resource as tainted and recreate it on the next apply. But 
+these issues (caused by invalid inputs, network disruptions, etc.) do not require resource recreation (ie. SDC re-installation) to resolve. 
+If one untaints this resource manually prior to applying again, this resource can start from where it left off and, if the cause of failure
+has been rectified, it can take incremental actions to set the necessary SDC parameters.
+So please untaint the resource before applying again if you want to prevent unnecessary SDC re-installations.
```
Comment (shenda1) [inline templates/resources/sdc_host.md.tmpl:34]: Can you provide the command for the same?

File: templates/resources/sdc_host.md.tmpl [hunk 3]
```diff
@@ -26,13 +26,20 @@ description: |-
 
 {{ .Description | trimspace }}
 
+~> **Caution:** SDC Host creation is not atomic. This resource sets parameters like name, etc. after SDC installation is complete.
+If that fails for any reason, Terraform, by default, will mark this resource as tainted and recreate it on the next apply. But
```
Comment (AnikaAgiwal2711) [inline templates/resources/sdc_host.md.tmpl:30]: Can we tell the reason here. we can mention here like : If that fails for any reason for eg: an sdc with same name already exists etc., then, Terraform.......

File: powerflex/provider/sdc_host_resource.go [hunk 4]
```diff
@@ -499,13 +498,10 @@ func (r *sdcHostResource) Update(ctx context.Context, req resource.UpdateRequest
 	}
 
 	// TODO: check that any the stuff that cannot be updated are not changed
```
Comment (Krunal-Thakkar) [inline powerflex/provider/sdc_host_resource.go:500]: remove TODO if completed

File: templates/resources/sdc_host.md.tmpl (hunk 2 above)
Comment (rounak-adhikary) [reply templates/resources/sdc_host.md.tmpl:34]: done

File: templates/resources/sdc_host.md.tmpl (hunk 3 above)
Comment (rounak-adhikary) [reply templates/resources/sdc_host.md.tmpl:30]: Its there in next line

File: powerflex/provider/sdc_host_resource.go (hunk 4 above)
Comment (rounak-adhikary) [reply powerflex/provider/sdc_host_resource.go]: done

**Review by Krunal-Thakkar — APPROVED** [review]
LGTM

**Comment by AnikaAgiwal2711** [comment]
run e2e test

**Review by AnikaAgiwal2711 — APPROVED** [review]

**Pull request merged**